types/humblebee__styled-components-breakpoint/index.d.ts

```diff
@@ -0,0 +1,66 @@
+// Type definitions for @humblebee/styled-components-breakpoint 2.1
+// Project: https://github.com/wearehumblebee/styled-components-breakpoint
+// Definitions by: Carl Ribbegårdh <https://github.com/CarlRibbegaardh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "@humblebee/styled-components-breakpoint" {
+    import { CSSObject, SimpleInterpolation } from "styled-components";
+
+    type Rule = "up" | "down" | "only";
+    interface Breakpoints  {
+        [key: string]: number;
+    }
+
+    // export const mediaWidthRule: (rule: Rule)
+    //     => string;
+    // export const ruleTemplate: (rule: string, width: number)
+    //     => string;
+    // export const mediaTemplate: (rules: string)
+    //     => string;
+    // export const getSmallestMedia: (breakpoints: Breakpoints)
+    //     => number;
+    // export const getNextMedia: (breakpoints: Breakpoints, width: number)
+    //     => number;
+    // export const mediaRules: (breakpoints: Breakpoints, widthKey: string, rule: Rule, boundKey: string)
+    //     => string;
+    // export const getMixin: (breakpoints: Breakpoints, key: string, rule?: Rule) => (bound?: string | undefined)
+    //     => (first: TemplateStringsArray | CSSObject, ...interpolations: SimpleInterpolation[])
+    //     => import("styled-components").FlattenSimpleInterpolation;
+    // export const getMediaShorthands: (breakpoints: Breakpoints, rule: Rule)
+    //     => {};
```

Comment:
^ mistake?

Reply:
While they are true, and might be beneficial for someone at some point, they are actually not used when using the library. They are internal from how I understand and use the library.

Reply:
https://www.npmjs.com/package/@humblebee/styled-components-breakpoint
They only use the default export in their documentation.